include/glow/Importer/Caffe2.h

```diff
@@ -38,6 +38,9 @@ class Value;
 /// Loads caffe2 models.
 class caffe2ModelLoader
     : public CommonOperatorLoader<caffe2::OperatorDef, caffe2::Argument> {
+  /// Get the broadcast attribute.
+  int getBroadcast(const ArgumentDictionaryTy &dict) override;
```

Comment:
I'd prefer to have Broadcast as `bool`. I think it is `int` in ONNX solely for storage purposes.